internal/options/flags.go

```diff
@@ -89,6 +89,13 @@ links to the repository for each code reference.`,
 		defaultValue: false,
 		usage: `If enabled, the scanner will terminate with exit code 0 when the
 LaunchDarkly API is unreachable or returns an unexpected response.`,
+	},
+	{
+		name:         "lookback",
+		short:        "l",
+		defaultValue: 10,
```

Comment:
This default added about 2 seconds to the overall execution time against one of our large repos on my local machine. As we test the preview version, we may want to tweak the default value if it turns out to be slow.